kokoro/config/test/third_party_apps/bookworm_x86_64.gcl

```diff
@@ -0,0 +1,9 @@
+import 'common.gcl' as common
+
+config build = common.third_party_apps_test {
+  params {
+    platforms = [
+      'debian-12',
+    ]
```

Comment:
```suggestion
    platforms = ['debian-12']
```

Reply:
these are autogenerated files, so that's why they are this way. Can we just leave them like this?

Reply:
I'm confused. If these are autogenerated, why are they inconsistent? Or are you saying that the others are manually written? If so, yes, we should leave them as-is and re-generate everything in a follow-up.

Reply:
The more recent ones are autogenerated by make_ops_agent_changes.go. the older ones have the all-on-one-line style because they were manually edited.

Reply:
Makes sense, let's do a consistency pass later.